typescript.html.markdown

```diff
@@ -16,7 +16,7 @@ This article will focus only on TypeScript extra syntax, as opposed to
 [JavaScript](/docs/javascript).
 
 To test TypeScript's compiler, head to the
-[Playground] (http://www.typescriptlang.org/Playground) where you will be able
+[Playground](http://www.typescriptlang.org/Playground) where you will be able
```

Comment:
I believe, now it's http://www.typescriptlang.org/play/, no?

Reply:
Great catch, it does redirect to `/play` now. I'll also use the `https` url as well.